ogcapi-stable/ogcapi-crs/src/main/java/de/ii/ogcapi/crs/domain/CrsConfiguration.java

```diff
@@ -56,6 +58,26 @@ abstract class Builder extends ExtensionConfiguration.Builder {}
    */
   Set<EpsgCrs> getAdditionalCrs();
 
+  /**
+   * @langEn If `true`, the coordinate reference systems will be included in every Collection
+   *     resource that is embedded in the Collections resource. The global `crs` array will not be
+   *     used or referenced. Use this option, if the API is intended to be used with a client that
+   *     does not support the global `crs` array.
+   * @langDe Bei `true` werden die Koordinatenreferenzsysteme in jede Collection-Ressource
+   *     aufgenommen, die in die Collections-Ressource eingebettet ist. Das globale `crs`-Array wird
+   *     nicht verwendet oder referenziert. Verwenden Sie diese Option, wenn die API mit Clients
+   *     verwendet werden soll, die das globale `crs`-Array nicht unterstützen.
+   * @default false
+   */
+  @Nullable
+  Boolean getSuppressGlobalCrsList();
+
+  @Value.Derived
+  @DocIgnore
+  default boolean shouldSuppressGlobalCrsList() {
+    return Boolean.TRUE.equals(getSuppressGlobalCrsList());
+  }
```

Comment:
It seems that `@DocIgnore` does not work correctly, this is visible in the docs. I will look at that, but in this case you should use `@JsonIgnore` anyway. 
`@DocIgnore` is only needed for very rare cases where an option should be read and/or written from/to json but should not show up in the docs.